README.md

```diff
@@ -72,7 +76,14 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+attr_accessor :name
+
+def friends
+  @friends = friends
+  @friends.each do |friend|
+    puts "Belle is friends with #{:name}"
+end
+
 ```
```

Comment:
close

`friends.each do |name|`
  `puts "Belle is friends with #{name}"`
`end`